<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_StatusA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_StatusB">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Thread A</string>
    </property>
    <widget class="QPushButton" name="btn_start_A">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pause_A">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_resume_A">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stopA">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop A</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Width</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Height</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_imgx">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1024</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_imgy">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1024</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame Rate(ms)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_framerate">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_low">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>20000</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_high">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>42000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Value</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>450</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Thread B</string>
    </property>
    <widget class="QPushButton" name="btn_start_B">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pause_B">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_resume_B">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stopB">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop B</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_histrate">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame Rate(ms)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>521</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="imgView">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>521</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="hist_chartview">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
